project.ruleset.xml.example

```diff
@@ -14,6 +14,9 @@
 	<!-- Exclude the Composer Vendor directory. -->
 	<exclude-pattern>/vendor/*</exclude-pattern>
 
+	<!-- Exclude minified Javascript files. -->
+	<exclude-pattern>*.min.js</exclude-pattern>
+
```

Comment:
Done & updated the PR description to reflect the changes.
(re-did the comit and removed the twig exclusion as well while I was at it)